packages/wonder-blocks-cell/src/components/internal/cell-core.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+
+import {CellMeasurements} from "./common.js";
+
+import type {CellProps, TypographyText} from "../../util/types.js";
+
+type CellCoreProps = {|
+    ...$Rest<CellProps, {|title: TypographyText|}>,
+
+    /**
+     * The content of the cell.
+     */
+    children: React.Node,
+|};
+
+/**
+ * CellCore is the base cell wrapper. It's used as the skeleton/layout that is
+ * used by BasicCell and DetailCell (and any other variants).
+ *
+ * Both variants share how they render their accessories, and the main
+ * responsibility of this component is to render the contents that are passed in
+ * (using the `children` prop).
+ */
+const CellCore = (props: CellCoreProps): React.Node => {
+    const {
+        children,
+        leftAccessory = undefined,
+        leftAccessoryStyle = undefined,
+        rightAccessory = undefined,
+        rightAccessoryStyle = undefined,
+    } = props;
+
+    return (
+        <View style={[styles.wrapper]}>
+            <View style={styles.innerWrapper}>
+                {/* Left accessory */}
+                {leftAccessory && (
+                    <>
+                        <View
+                            style={[styles.accessory, {...leftAccessoryStyle}]}
+                        >
+                            {leftAccessory}
+                        </View>
+                        <Strut
+                            size={CellMeasurements.accessoryHorizontalSpacing}
+                        />
+                    </>
+                )}
+                {/* Cell contents */}
+                <View style={styles.content}>{children}</View>
+
+                {/* Right accessory */}
+                {rightAccessory && (
+                    <>
+                        <Strut
+                            size={CellMeasurements.accessoryHorizontalSpacing}
+                        />
+                        <View
+                            style={[
+                                styles.accessory,
+                                styles.accessoryRight,
+                                {...rightAccessoryStyle},
+                            ]}
+                        >
+                            {rightAccessory}
+                        </View>
+                    </>
+                )}
+            </View>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    wrapper: {
+        background: Color.white,
```

Comment:
yep, that's a conversation that we had with Reony and it would make sense to default to white so the inner wrapper can use white as the base color in case we want to pass in backgrounds with alpha (e.g. with the `selected / active` state). @megaroeny in case you want to add more context on this.

Reply:
Yup agreed!